src/main/java/org/codeforamerica/shiba/configurations/FilterConfig.java

```diff
@@ -27,9 +27,7 @@ public FilterConfig(SessionLogFilter sessionLogFilter) {
         "/submit",
         "/submit-feedback",
         "/submit-documents",
-        "/download-caf/*",
-        "/download-ccap/*",
-        "/download-certain-pops/*"
+        "/download**"
```

Comment:
Includes /download, download-caf/*, /download-ccap, download-ccap/* and any other endpoints we add that start with "download"